soccer/joystick/GamepadJoystick.cpp

```diff
@@ -1,37 +1,33 @@
 #include "GamepadJoystick.hpp"
```

Comment:
Can this file and GampadJoystic.hpp be removed? (It dosent seem to be used anymore).